Bayes_Rules/Chapter_4_Balance_and_Sequentiality_in_Bayesian_Analyses.ipynb

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
<p>i believe this file can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc_resources/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>